core/src/main/scala/caliban/parsing/Parser.scala

```diff
@@ -266,6 +268,62 @@ object Parser {
         )
     }
 
+  private def schemaExtension[_: P]: P[SchemaExtension] =
+    P("extend schema" ~/ directives.? ~ "{" ~ rootOperationTypeDefinition.rep ~ "}").map {
```

Comment:
It should be possible to extend a schema and only provide directives without any operation: 
http://spec.graphql.org/June2018/#sec-Schema-Extension (2nd case)